app/client/ts/video-model.ts

```diff
@@ -276,7 +276,7 @@ export default class Video implements Drop {
         const comment = await VideoComment.new(
             owner,
             text,
-            new Date().getTime(),
+            new Date(),
```

Comment:
We now just save the Date as a Date in case we need the functionality later.

I just realized that if we choose to just save the number, we can always recreate the Date object later as well so if you'd like to change this back, I'm game.